source/slang/slang-parser.cpp

```diff
@@ -3882,6 +3883,14 @@ namespace Slang
         return decl;
     }
 
+    static NodeBase* parseIfLetDecl(
+        Parser* parser, void* /*userData*/)
```

Comment:
This is not needed, you can just parse the fields directly into IfLetStmt.

Reply:
done